### ISSUE TITLE
Looks like authentication is still required for the appimage package

### ISSUE THREAD

**Issue by cgarwood82** [sourceryltd/makerhub-releases#1]
Running latest arch linux and attempting to run the latest appimage, the output from the commandline looks like it's still looking for an authenticated user. 

```
Fontconfig warning: "/usr/share/fontconfig/conf.avail/05-reset-dirs-sample.conf", line 6: unknown element "reset-dirs"
2023-09-23T20:11:47.331Z - info init Loading configuration from "/home/trans/.makerverse"
2023-09-23T20:11:47.649Z - info init Starting the server at http://127.0.0.1:46181
2023-09-23T20:11:49.999Z - warn access Forbidden: ipaddr=127.0.0.1, code="undefined", message="No token provided"
2023-09-23T20:11:50.118Z - warn access Forbidden: ipaddr=127.0.0.1, code="undefined", message="No token provided"
2023-09-23T20:11:50.120Z - warn access Forbidden: ipaddr=127.0.0.1, code="undefined", message="No token provided"
2023-09-23T20:11:50.132Z - warn access Error: No token provided
    at validateToken (/tmp/.mount_MakerHGwMyXV/resources/app/server/index.js:3905:13)
    at Array.authSocketIoMiddleware (/tmp/.mount_MakerHGwMyXV/resources/app/server/index.js:3946:19)
    at run (/tmp/.mount_MakerHGwMyXV/resources/app/node_modules/socket.io/lib/namespace.js:124:11)
    at Namespace.run (/tmp/.mount_MakerHGwMyXV/resources/app/node_modules/socket.io/lib/namespace.js:136:3)
    at Namespace.add (/tmp/.mount_MakerHGwMyXV/resources/app/node_modules/socket.io/lib/namespace.js:164:8)
    at Client.doConnect (/tmp/.mount_MakerHGwMyXV/resources/app/node_modules/socket.io/lib/client.js:97:20)
    at Client.connect (/tmp/.mount_MakerHGwMyXV/resources/app/node_modules/socket.io/lib/client.js:66:17)
    at Server.onconnection (/tmp/.mount_MakerHGwMyXV/resources/app/node_modules/socket.io/lib/index.js:435:10)
    at Server.emit (events.js:182:13)
    at Server.handshake (/tmp/.mount_MakerHGwMyXV/resources/app/node_modules/engine.io/lib/server.js:338:8)
    at /tmp/.mount_MakerHGwMyXV/resources/app/node_modules/engine.io/lib/server.js:232:12
    at Server.checkRequest (/tmp/.mount_MakerHGwMyXV/resources/app/node_modules/socket.io/lib/index.js:76:51)
    at Server.verify (/tmp/.mount_MakerHGwMyXV/resources/app/node_modules/engine.io/lib/server.js:168:17)
    at Server.handleRequest (/tmp/.mount_MakerHGwMyXV/resources/app/node_modules/engine.io/lib/server.js:222:8)
    at Server.<anonymous> (/tmp/.mount_MakerHGwMyXV/resources/app/node_modules/engine.io/lib/server.js:466:14)
    at Server.<anonymous> (/tmp/.mount_MakerHGwMyXV/resources/app/node_modules/socket.io/lib/index.js:343:16)
2023-09-23T20:11:50.184Z - warn api:users Login Failure Result { errors:
   [ { message:
        'The current user is not authorized to access this resource.',
       type: 'Exception',
       code: 'AUTH_NOT_AUTHENTICATED',
       locations: [Array],
       path: [Object] } ],
  meta: { traceId: '0HMTJ5H6ULGRN:00000002' } }
2023-09-23T20:11:50.184Z - warn api:users Login Error User not found
(node:3029) UnhandledPromiseRejectionWarning: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:470:11)
    at ServerResponse.header (/tmp/.mount_MakerHGwMyXV/resources/app/node_modules/express/lib/response.js:767:10)
    at ServerResponse.send (/tmp/.mount_MakerHGwMyXV/resources/app/node_modules/express/lib/response.js:170:12)
    at ServerResponse.json (/tmp/.mount_MakerHGwMyXV/resources/app/node_modules/express/lib/response.js:267:15)
    at ServerResponse.send (/tmp/.mount_MakerHGwMyXV/resources/app/node_modules/express/lib/response.js:158:21)
    at /tmp/.mount_MakerHGwMyXV/resources/app/server/index.js:3524:9
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:3029) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:3029) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

```

User experience from my end when setting up my M2 is that upon selecting the sled size, it immediately says "Connecting" and does nothing else.

**Comment by kevinjohnson123** [comment]
i haven't tested this on Arch linux,  can you send me a  Link to an ISO to the Arch linux install CD you're using. I'll setup Arch linux in a VM  and see if i can replicate the issue.
Testing was previously done on  Win7, macOS and  Linux Mint.

**Comment by cgarwood82** [comment]
ISO install for arch can be challenging as most of the installation is manual since there isn’t a graphical installer that I’m aware of. You could use a Manjaro installer which is based on arch and should be a pretty good testing environment. 

https://manjaro.org/download/

**Comment by kevinjohnson123** [comment]
> ISO install for arch can be challenging as most of the installation is manual since there isn’t a graphical installer that I’m aware of. You could use a Manjaro installer which is based on arch and should be a pretty good testing environment.
> 
> https://manjaro.org/download/

Fixed in  makerhub  v1.3.1
